test/blocklist.json

```diff
@@ -1,5 +1,5 @@
 {
   "Linux": ["test-multi-nodes.js", "test-msg-type-py-node.js", "test-msg-type-cpp-node.js", "test-cross-lang.js"],
   "Darwin": ["test-multi-nodes.js", "test-msg-type-py-node.js", "test-msg-type-cpp-node.js", "test-cross-lang.js"],
-  "Windows_NT": ["test-multi-nodes.js", "test-msg-type-py-node.js", "test-msg-type-cpp-node.js", "test-cross-lang.js"]
+  "Windows_NT": ["test-multi-nodes.js", "test-msg-type-py-node.js", "test-msg-type-cpp-node.js", "test-cross-lang.js", "test-generate-messages-bin.js"]
```

Comment:
@wayneparrott I disable `test-generate-messages-bin.js` temporarily on Windows because it downloads rclnodejs 0.18.0 which failed with nodejs 10. We can add it back when we have a new release of rclnodejs.